src/seedu/addressbook/commands/Tagging.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	
+	private String updatedTag;
+	
+	private String prepareTagForAddition(boolean add, String name, Tag tag){
```

Comment:
- You if-else brackets are not at all in the standard we expect. Please check the standard and take note NOT to do this mistake again.

- indentation is a bit off too?